tasks/users.yml

```diff
@@ -0,0 +1,64 @@
+---
+- block:
+    - name: Create tmpfile
+      tempfile:
```

Comment:
Might be cleaner (and arguably more secure) to use [tempfile](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/tempfile_module.html) for this.

Reply:
I'm not sure I follow? The tempfile module is being used here.

Reply:
Apologies, I was looking at something else and got confused - you're right of course.